python/GangaDirac/Lib/RTHandlers/DiracRTHUtils.py

```diff
@@ -87,7 +87,7 @@ def dirac_outputfile_jdl(output_files, empty_SE_check):
             if empty_SE_check:
                 ## If true check, if not false check
                 raise BackendError("Dirac", "Can't submit a DIRAC job with DiracFile outputfile without setting a defaultSE.")
-            config = getConfig('Dirac')
+            config = getConfig('DIRAC')
```

Comment:
If you look at the console output of the test, at the end it has:
```
Project Ganga v602r1 is not available for platform x86_64-slc6-gcc49-opt
Build step 'Virtualenv Builder' marked build as failure
```

I'm guessing this is because 6.2.1 was removed for LHCb?